### PR TITLE
document parse method

### PR DESCRIPTION
**Pull request by HenrikJoreteg**
like it says on the box :)

### CODE REVIEW

**Comment by kamilogorek** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -229,6 +229,33 @@ console.log(JSON.stringify(collection));
...
+  "offset": 0,
+  "data": [
+    {"name": "larry"},
+    {"name": "curly"},
+    {"name": "moe"}
+  ]
+}
+```
+
+To extract `results` you'd define a `parse` method on the collection as follows, to return the array of data to be stored.
```
Comment (aaronmccall) [inline README.md]: Should \`results\` be \`data\` here?

**Comment by HenrikJoreteg** [comment]
good catch @aaronmccall, fixed it.

**Comment by wraithgar** [comment]
Oh hello there we should merge this thanks for your patience.

**Comment by wraithgar** [comment]
the site uses npm to pull docs so this is a patch release in order for it to show up on the site.

**Comment by wraithgar** [comment]
published as `v1.6.1`

**Pull request merged**